ONScripterReporter.h

```diff
@@ -0,0 +1,22 @@
+#include "ONScripterLabel.h"
+#include "Reporter.h"
+
+#ifndef _ONSCRIPTERREPORTER_H
+#define _ONSCRIPTERREPORTER_H
+
+class ONScripterReporter : public Reporter {
+public:
+    ONScripterReporter(ONScripterLabel *label) {
+        this->label = label;
+    };
+
+    ~ONScripterReporter() {
+        this->label = NULL;
+    };
+
+    bool reportError( const char *title, const char *errstr, bool is_simple=false, bool is_warning=false ) const;
+private:
+    ONScripterLabel *label;
```

Comment:
Maybe `onscripter`? For all intents and purposes this is basically our version of an "engine", and despite that it's part of the class name, "label" doesn't really get it's purpose across.

I still wonder why it was called ONScripterLabel, maybe I'm missing something?

Reply:
I'm OK with renaming that class in this PR if there's no concerns about doing so from @Galladite27 .

Reply:
Oh I just meant the variable! I'd _love_ to rename the class, but that seems like a much bigger change, and one to think on regarding a better name. It's also more likely to cause (small) problems when wanting to downstream code from insani.

Reply:
Renamed the variable, no problem.

Reply:
There's no need to change the class' name. It works fine, both Seung and I are used to it, and changing it provides nothing. This isn't some big project where we need to worry about newbie contributors getting confused - keeping things the same and just learning how they work is far preferable here.

Reply:
Playmer's point is also valid - potentially giving ourselves (read: Seung) more work when porting code over a name change is ridiculous.